ngraph/test/util/engine/interpreter_engine.cpp

```diff
@@ -116,8 +116,8 @@ test::INTERPRETER_Engine::INTERPRETER_Engine(const std::shared_ptr<Function> fun
     m_executable = m_backend->compile(m_function);
     for (auto i = 0; i < m_function->get_output_size(); ++i)
     {
-        m_result_tensors.push_back(m_backend->create_tensor(m_function->get_output_element_type(i),
-                                                            m_function->get_output_shape(i)));
+        m_result_tensors.push_back(m_backend->create_dynamic_tensor(
```

Comment:
@itikhono We have dedicated constructor to support dynamic tensors (https://github.com/openvinotoolkit/openvino/blob/bc346c3aeff8ce914b570d70c16e42ff7210c91c/ngraph/test/util/engine/interpreter_engine.cpp#L132). In order to enable it, `TestCase` should be created with `TestCaseType::DYNAMIC` argument (`auto test_case = test::TestCase<TestEngine, TestCaseType::DYNAMIC>(function);`)
I can do it in my PR with Loop tests enablement, but please revert this change.

Reply:
fixed